### PR TITLE
Update connect-to-microsoft-365-powershell.md

### PR DESCRIPTION
**Pull request by msbemba**
Added a section for connecting to Microsoft Graph powershell as powershell graph SDK is going to replace AzureAD module  in the future.

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/8299

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/enterprise/connect-to-microsoft-365-powershell.md [hunk 1]
```diff
@@ -200,6 +200,13 @@ You will need an active Azure subscription for your organization that is tied to
 
 For more information, see [Azure Cloud Shell](/azure/cloud-shell/overview).
 
+
+## Get started with the Microsoft Graph PowerShell SDK
+
+You can use Microsoft Graph PowerShell SDK to Access to all Microsoft Graph APIs.
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/connect-to-microsoft-365-powershell.md]:
```suggestion
You can use the Microsoft Graph PowerShell SDK to Access all Microsoft Graph APIs.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

File: microsoft-365/enterprise/connect-to-microsoft-365-powershell.md [hunk 2]
```diff
@@ -200,6 +200,13 @@ You will need an active Azure subscription for your organization that is tied to
 
 For more information, see [Azure Cloud Shell](/azure/cloud-shell/overview).
 
+
+## Get started with the Microsoft Graph PowerShell SDK
+
+You can use the Microsoft Graph PowerShell SDK to Access all Microsoft Graph APIs.
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/connect-to-microsoft-365-powershell.md]:
```suggestion
You can use the Microsoft Graph PowerShell SDK to access all Microsoft Graph APIs.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Apoplogies @msbemba, one more, please.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@kelleyvice-msft Please merge this PR. Thanks

**Pull request merged**